File: POS/POS/InventoryAddQuantityForm.vb
Imports MySql.Data.MySqlClient

Public Class InventoryAddQuantityForm
    Dim initialquantity As Integer
    Dim conn As New MySqlConnection
    Dim connstring As String = "Server=localhost;User Id=root;Password=;oDatabase=POS"

    Private Sub Button2_Click(sender As Object, e As EventArgs) Handles Button2.Click
        Me.Close()
    End Sub

    Private Sub InventoryAddQuantityForm_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        initialquantity = InventoryForm.TextBox3.Text
        Me.Label2.Text = InventoryForm.TextBox2.Text
        Me.Label3.Text = InventoryForm.Label8.Text
    End Sub

    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        Try
            Dim dummy As Integer = CInt(TextBox1.Text)
        Catch ex As Exception
            MsgBox("Please enter a valid quantity")
            Exit Sub
        End Try
        Try
            conn.ConnectionString = connstring
            conn.Open()

            Dim finalquantity As Integer = initialquantity + CInt(TextBox1.Text)

            Dim comm As New MySqlCommand
            Dim query As String = "UPDATE producttbl SET productquantity = " & finalquantity & " WHERE productid = '" & Label3.Text & "'"
            'UPDATE `producttbl` SET `productid`=[value-1],`productname`=[value-2],`productcategoryid`=[value-3],`productquantity`=[value-4],`productprice`=[value-5] WHERE 1

            comm.CommandText = query
            comm.Connection = conn
            comm.ExecuteNonQuery()
        Catch ex As Exception
            MsgBox(ex.ToString)
            Exit Sub
        Finally
            conn.Close()
        End Try

        Try
            conn.ConnectionString = connstring
            conn.Open()

            Dim comm As New MySqlCommand
            Dim query As String = "INSERT INTO logstbl(username,process,description,dateandtime) VALUES('" & LoginForm.username & "','Inventory - Add Quantity','Quantity (" & TextBox1.Text & ") added to Product (" & Label2.Text & ")',NOW())"

            comm.Connection = conn
            comm.CommandText = query
            comm.ExecuteNonQuery()
        Catch ex As Exception
            MsgBox(ex.ToString)
        Finally
            conn.Close()
        End Try

        InventoryForm.searchitem(InventoryForm.TextBox1.Text, InventoryForm.ComboBox1.Text)
        InventoryForm.iteminfo(Label3.Text)
        Me.Close()

    End Sub
End Class

File: ACE Hardware/ACE Hardware/InventoryAddQuantityForm.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;
using MySql.Data.MySqlClient;

namespace ACE_Hardware
{
    public partial class InventoryAddQuantityForm : Form
    {

        private int initialquantity;
        private MySqlConnection conn = new MySqlConnection();
        private string connstring = Program.connstring;

        public InventoryAddQuantityForm()
        {
            InitializeComponent();
        }

        private void Button2_Click(object sender, EventArgs e)
        {
            this.Close();
        }

        private void InventoryAddQuantityForm_Load(object sender, EventArgs e)
        {
            initialquantity = int.Parse(Program.inventoryFormInstance.TextBox3.Text);
            this.Label2.Text = Program.inventoryFormInstance.TextBox2.Text;
            this.Label3.Text = Program.inventoryFormInstance.Label8.Text;
        }

        private void Button1_Click(object sender, EventArgs e)
        {
            try
            {
                int dummy = System.Convert.ToInt32(TextBox1.Text);
            }
            catch (Exception ex)
            {
                MessageBox.Show("Please enter a valid quantity");
                return;
            }
            try
            {
                conn.ConnectionString = connstring;
                conn.Open();

                int finalquantity = initialquantity + System.Convert.ToInt32(TextBox1.Text);

                MySqlCommand comm = new MySqlCommand();
                string query = "UPDATE producttbl SET productquantity = " + finalquantity + " WHERE productid = '" + Label3.Text + "'";
                // UPDATE `producttbl` SET `productid`=[value-1],`productname`=[value-2],`productcategoryid`=[value-3],`productquantity`=[value-4],`productprice`=[value-5] WHERE 1

                comm.CommandText = query;
                comm.Connection = conn;
                comm.ExecuteNonQuery();
            }
            catch (Exception ex)
            {
                MessageBox.Show(ex.ToString());
                return;
            }
            finally
            {
                conn.Close();
            }

            try
            {
                conn.ConnectionString = connstring;
                conn.Open();

                MySqlCommand comm = new MySqlCommand();
                string query = "INSERT INTO logstbl(username,process,description,dateandtime) VALUES('" + Program.login.username + "','Inventory - Add Quantity','Quantity (" + TextBox1.Text + ") added to Product (" + Label2.Text + ")',NOW())";

                comm.Connection = conn;
                comm.CommandText = query;
                comm.ExecuteNonQuery();
            }
            catch (Exception ex)
            {
                MessageBox.Show(ex.ToString());
            }
            finally
            {
                conn.Close();
            }

            Program.inventoryFormInstance.searchitem(Program.inventoryFormInstance.TextBox1.Text, Program.inventoryFormInstance.ComboBox1.Text);
            Program.inventoryFormInstance.iteminfo(Label3.Text);
            this.Close();
        }
    }
}
